### PR TITLE
Add ‘moji’ types

### PR DESCRIPTION
**Pull request by yasupeke**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/moji/tslint.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-irregular-whitespace": false
```
Comment (plantain-00) [inline types/moji/tslint.json]: For new package types, it should not include other rules

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/moji/tslint.json (hunk 1 above)
Comment (yasupeke) [reply types/moji/tslint.json]: moji-tests.ts
```
moji('　').convert('ZS', 'HS').toString();
```
This test code causes an error.

Comment (plantain-00) [reply types/moji/tslint.json]: You can change to below code to just disable the rule:
```
// tslint:disable-next-line:no-irregular-whitespace
moji('　').convert('ZS', 'HS').toString();
```

**Review by plantain-00 — APPROVED** [review]

File: types/moji/tslint.json (hunk 1 above)
Comment (yasupeke) [reply types/moji/tslint.json]: You really saved me.

**Pull request merged**